src/tools/object_selector.cpp

```diff
@@ -161,19 +154,22 @@ bool ObjectSelector::selectBox(const MapCoordF& corner1, const MapCoordF& corner
 }
 
 
-bool ObjectSelector::sortObjects(const std::pair< int, Object* >& a, const std::pair< int, Object* >& b)
+void ObjectSelector::sortObjectSelection(SelectionInfoVector& objects)
 {
-	if (a.first != b.first)
-		return a.first < b.first;
-	
-	auto a_area = a.second->getExtent().width() * a.second->getExtent().height();
-	auto b_area = b.second->getExtent().width() * b.second->getExtent().height();
-	
-	return a_area < b_area;
+	std::sort(objects.begin(), objects.end(), 
+		[](const std::pair<int, Object*>& a, const std::pair<int, Object*>& b) {
+			if (a.first != b.first)
+				return a.first < b.first;
+			
+			auto a_area = a.second->getExtent().width() * a.second->getExtent().height();
+			auto b_area = b.second->getExtent().width() * b.second->getExtent().height();
+			return a_area < b_area;
+		}
+	);
 }
```

Comment:
We could have done that from the beginning, but we didn't. Embrace `std::sort` and the custom operator, don't add `sortObjectSelection`.